lametro/search_indexes.py

```diff
@@ -50,8 +51,18 @@ def prepare_attachment_text(self, obj):
         )
 
     def prepare_legislative_session(self, obj):
-        start_year = obj.legislative_session.identifier
-        end_year = int(start_year) + 1
+        if len(obj.actions_and_agendas) > 0:
+            most_recent = sorted(obj.actions_and_agendas, key=lambda i: i['date'],reverse=True)[0]
+            action_date = most_recent['event'].start_time
+        else:
+            action_date = obj.get_last_action_date()
```

Comment:
`obj.actions_and_agendas` should include all actions, so this method will return `None`, which will cause the subsequent logic which assumes a date to fail. If there are no actions or agendas, we should return `None` here.

```suggestion
        else:
            return None
```